### PR TITLE
CT-1118 Add and test import with importAsNull

### PR DESCRIPTION
**Pull request by tomasfejfar**
Jira: CT-1118
Connection PR: https://github.com/keboola/connection/pull/4902
SAPI PR: https://github.com/keboola/storage-api-php-client/pull/1296
StorageBackend PR: https://github.com/keboola/storage-backend/pull/136
Driver PR: https://github.com/keboola/php-storage-driver-bigquery/pull/123

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

image

### CODE REVIEW

File: tests/Backend/CommonPart1/ImportExportCommonTest.php [hunk 1]
```diff
@@ -404,6 +405,21 @@ public function testTableAsyncImportMissingFile(): void
         }
     }
 
+    public function testImportAsNull(): void
```
Comment (jirkasemmler) [inline tests/Backend/CommonPart1/ImportExportCommonTest.php:408]: udelej tady prosim jeste test na to, kdyz tam posles vice hodnot v `importAsNull`

File: apiary.apib [hunk 2]
```diff
@@ -9572,6 +9572,7 @@ Requesting changes will remove all approvals and move the Merge request to `deve
 ### CreateTableDefinitionAsyncFile(CreateTableDefinitionAsyncBase)
 
 + dataFileId (required, number) - Id of the file stored in [File Uploads](#reference/files)
++ importAsNull (optional, array[string]) - (Snowflake and BigQuery only) Array of string values that should be imported as NULL values. The default value is `['']`. Some other common values you might consider are `'\N'` or `'null'`. Multiple values allowed for Snowflake backend. For BigQuery only single value is allowed and others are ignored.
```
Comment (jirkasemmler) [inline apiary.apib:9575]: dopln sem prosim, ze `importAsNull` muze mit jen 1 hodnotu zatim

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
ok, jen prosim dopln tu limitaci na 1 hodnotu , na ktere jsme se shodli tady https://github.com/keboola/connection/pull/4902#discussion_r1533450606

File: apiary.apib (hunk 2 above)
Comment (tomasfejfar) [reply apiary.apib]: Done

File: tests/Backend/CommonPart1/ImportExportCommonTest.php (hunk 1 above)
Comment (tomasfejfar) [reply tests/Backend/CommonPart1/ImportExportCommonTest.php:408]: Done

File: tests/Backend/CommonPart1/ImportExportCommonTest.php [hunk 3]
```diff
@@ -404,6 +405,33 @@ public function testTableAsyncImportMissingFile(): void
...
+    {
+        $filePath = __DIR__ . '/../../_data/languages.csv';
+        $importFile = new CsvFile($filePath);
+
+        // test invalid option
+        try {
```
Comment (jirkasemmler) [inline tests/Backend/CommonPart1/ImportExportCommonTest.php:414]: ajajaj, baad boy :)

**Review by jirkasemmler — APPROVED** [review]
jsem ti fixnul ten stan, za me ok :)

**Pull request merged**